docs/static_site/src/_config.yml

```diff
@@ -37,6 +37,18 @@ description: >- # this means to ignore newlines until "baseurl:"
 twitter_username: apachemxnet
 github_username:  apache/incubator-mxnet
 youtube_username: apachemxnet
+default_version: 1.6
```

Comment:
Don't we have the versions as input parameter for the Jenkins job? Can't we just use those values? The fewer duplications, the better.

Reply:
Removed this `default_version` key, while it is only used for version drop down default selection and it could always be the newest version below master.